### PR TITLE
Updated homepage styling

### PR DESCRIPTION
**Pull request by theswerd**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here:
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?

- Update Homepage styling
- Added donate button and app version notes at bottom
- created pageScaffold class

## Did you add any dependencies?

No
## How did you test the change?

iOS simulator

Simulator Screen Shot - iPhone 8 Plus - 2020-03-29 at 18 19 19
Simulator Screen Shot - iPhone 8 Plus - 2020-03-29 at 18 19 26
Simulator Screen Shot - iPhone 8 Plus - 2020-03-29 at 18 19 30

### CODE REVIEW

File: client/flutter/lib/pages/home_page.dart [hunk 1]
```diff
@@ -54,93 +55,141 @@ class _HomePageState extends State<HomePage> {
...
-        padding: const EdgeInsets.symmetric(horizontal: 15.0),
-        child: SafeArea(
-          child: CustomScrollView(slivers: [
-            SliverAppBar(
-                expandedHeight: 110,
-                backgroundColor: Colors.white,
-                flexibleSpace: Image.asset("assets/WHO.jpg")),
-            SliverStaggeredGrid.count(
+    return PageScaffold(context,
+        title: "Corona Virus",
```
Comment (advayDev1) [inline client/flutter/lib/pages/home_page.dart]: where is this title from with a space?

File: client/flutter/lib/pages/home_page.dart [hunk 2]
```diff
@@ -54,93 +55,141 @@ class _HomePageState extends State<HomePage> {
...
+                ),
+              ),
+              ListTile(
+                leading: Icon(Icons.share),
+                title: Text(S.of(context).homePagePageSliverListShareTheApp),
+                trailing: Icon(Icons.arrow_forward_ios),
+                onTap: () => Share.share(
+                    S.of(context).commonWhoAppShareIconButtonDescription),
+              ),
+              /*UNCOMMENT WHEN ADDING SETTINGS TILE
```
Comment (advayDev1) [inline client/flutter/lib/pages/home_page.dart]: just delete this

File: client/flutter/lib/pages/home_page.dart [hunk 3]
```diff
@@ -54,93 +55,141 @@ class _HomePageState extends State<HomePage> {
...
+              ListTile(
+                title: Text(S.of(context).homePagePageSliverListAboutTheApp),
+                trailing: Icon(Icons.arrow_forward_ios),
+                onTap: () => showAboutDialog(
+                    context: context,
+                    applicationLegalese:
+                        S.of(context).homePagePageSliverListAboutTheAppDialog),
+              ),
+              Container(height: 25,),
+              Text("Version 0.1 (12412)\n© 2020 WHO", style: TextStyle(color: Color(0xff26354E)), textAlign: TextAlign.center,),
```
Comment (advayDev1) [inline client/flutter/lib/pages/home_page.dart]: todo to pull these values in

File: client/flutter/lib/pages/home_page.dart [hunk 4]
```diff
@@ -54,93 +55,141 @@ class _HomePageState extends State<HomePage> {
...
+                  padding: EdgeInsets.symmetric(vertical:24, horizontal:23),
+                  color: Color(0xffCA6B35),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: <Widget>[
+                      Text("Donate here"),
+                      Icon(Icons.arrow_forward_ios)
+                    ],
+                  ),
+                  onPressed: ()=>launch("https://www.who.int/Covid19ResponseFund")//TODO: DOUBLE CHECK LINK
```
Comment (advayDev1) [inline client/flutter/lib/pages/home_page.dart]: yes this is right, remove todo

File: client/flutter/lib/pages/home_page.dart (hunk 1 above)
Comment (theswerd) [reply client/flutter/lib/pages/home_page.dart]: The Figma Design

File: client/flutter/lib/pages/home_page.dart (hunk 3 above)
Comment (theswerd) [reply client/flutter/lib/pages/home_page.dart]: @advayDev1 What values?

Comment (theswerd) [reply client/flutter/lib/pages/home_page.dart]: What values?

**Pull request merged**